cmake/target/src/target_config.c

```diff
@@ -63,11 +64,11 @@ Target_CfeConfigData GLOBAL_CFE_CONFIGDATA =
       .SystemMain = CFE_ES_Main,
       .SystemExceptionISR = CFE_ES_ProcessCoreException,
 
-      /*
+      /* TODO: remove, or replace with the name of the table?
```

Comment:
I think its OK.  Even when loading from a table one still needs the actual table filename, right?